libs/designer/src/lib/ui/panel/recommendation/recommendationPanelContext.tsx

```diff
@@ -86,6 +89,7 @@ export const RecommendationPanelContext = (props: RecommendationPanelContextProp
   const navigateBack = useCallback(() => {
```

Comment:
warning for missing dependency (i don't know why github doesn't let me put the comment on line 87)